reddit/subreddit.py

```diff
@@ -35,24 +34,36 @@ def get_subreddit_threads():
         username=os.getenv("REDDIT_USERNAME"),
         password=passkey,
     )
+    # If the user inserts a thread link, pick that one
+    if os.getenv("THREAD_LINK"):
```

Comment:
if os.getenv("THREAD_LINK"):

makes it look for THREAD_LINK in .env not THREAD